addons/sale_stock/models/sale_order_line.py

```diff
@@ -219,11 +219,6 @@ def write(self, values):
         if 'product_uom_qty' in values:
             lines = self.filtered(lambda r: r.state == 'sale' and not r.is_expense)
 
-        if 'product_packaging_id' in values:
```

Comment:
Wouldn't it be nice to update the move `package_uom_id` if the selected uom in the `sale.order.line` is different than the one used in the move's uom?

Though it depends if we make the `sale.order.line` uom editable once the SO is confirmed or not. Before it was possible to still edit the packaging but not the uom, now it's impossible to edit the uom (which serves as an packaging too)

Reply:
It should be updated automatically with this compute: https://github.com/odoo/odoo/pull/184131/files#diff-2b9de2e50ff5e1dc0362b825bac2b07623770fb3275b3257ef972f255f3ccb8bR25-R31.